packages/nimble-components/src/table-column/date-text/group-header-view/index.ts

```diff
@@ -17,23 +18,19 @@ export class TableColumnDateTextGroupHeaderView extends TableColumnTextGroupHead
 TableNumberFieldValue,
 TableColumnDateTextColumnConfig
 > {
-    private static readonly formatter = new Intl.DateTimeFormat(undefined, {
-        dateStyle: 'medium',
-        timeStyle: 'medium'
-    });
+    private columnConfigChanged(): void {
+        TableColumnDateTextCellView.updateFormatter(this.columnConfig);
+        this.updateText();
+    }
 
     private groupHeaderValueChanged(): void {
-        if (typeof this.groupHeaderValue === 'number') {
-            try {
-                this.text = TableColumnDateTextGroupHeaderView.formatter.format(
-                    this.groupHeaderValue
-                );
-            } catch (e) {
-                this.text = '';
-            }
-        } else {
-            this.text = '';
-        }
+        this.updateText();
+    }
+
+    private updateText(): void {
+        this.text = TableColumnDateTextCellView.formatNumericDate(
```

Comment:
I expect this to be reworked at least a little bit based on my comment regarding the static formatter of the cell view causing problems, but I'll make a note here as well -- it doesn't seem quite right to me for the group header to be calling into the cell to do formatting for it. I think it would be better to have shared logic in an independent location that they both could call to avoid code duplication.